.github/workflows/build.yml

```diff
@@ -44,6 +44,7 @@ jobs:
         run: echo "@scientist-softserv:registry=https://npm.pkg.github.com" >> $NPM_CONFIG_USERCONFIG
       - run: yarn install
       - run: yarn test
+      - run: npm install next-auth
```

Comment:
should we mix `yarn` and `npm` commands together?

also, next-auth is already in the "yarn.lock" file, so it should be installed with the `yarn install` command above. I'd think the install goes fine because we are able to sign in.

Reply:
This was just a test that failed. I did finally see it was already installed by yarn, but thank you for checking in with me on this :)